### ISSUE TITLE
Cannot Use sbt-mu-scrgen and scalafix 0.11.1

### ISSUE THREAD

**Issue by collinkueter** [higherkindness/sbt-mu-srcgen#417]
Is there anyone looking into upgrading so the latest version of scala fix can be used? 

```
[error] 	* ch.epfl.scala:scalafix-interfaces:0.11.1 (early-semver) is selected over {0.10.4}
[error] 	    +- ch.epfl.scala:sbt-scalafix:0.11.1 (sbtVersion=1.0, scalaVersion=2.12) (depends on 0.11.1)
[error] 	    +- io.higherkindness:mu-srcgen-core_2.12:0.30.3       (depends on 0.10.4)
[error] 	    +- ch.epfl.scala:scalafix-cli_2.12.17:0.10.4          (depends on 0.10.4)
```

**Comment by fedefernandez** [comment]
@collinkueter, thanks for submitting the issue. The upgrade PR is a bit broken at this moment. I'll try to find some time to see if I can fix it. https://github.com/higherkindness/sbt-mu-srcgen/pull/412

**Comment by fedefernandez** [comment]
This should be fixed with the latest release (`0.31.0`)

**Comment by collinkueter** [comment]
Thank you!
